src/pages/en/core-concepts/routing.md

```diff
@@ -187,6 +149,55 @@ const { title, text } = Astro.props;
 </html>
 ```
 
+### Server (SSR) Mode
+In [SSR mode](/en/guides/server-side-rendering/), dynamic routes are defined the same way: include `[param]` or `[...path]` brackets in your file names to match arbitrary strings or paths. But because the routes are no longer built ahead of time, the page will be served to any matching route, and `getStaticPaths` has no effect. 
+
+```astro title="src/pages/resources/[resource]/[id].astro"
+---
+const { resource, id } = Astro.params;
+---
+<h1>{resource}: {id}<h1>
+```
+This page will be served for any value of `resource` and `id`: `resources/users/1`, `resources/colors/blue`, etc.
+
+#### Modifying the `[...slug]` example for SSR
+
+Because SSR pages can't use `getStaticPaths`, they can't receive props. Here, we modify our [earlier example](#example-dynamic-pages-at-multiple-levels) to work in SSR by looking up the value of the `slug` param in an object.
```

Comment:
Given folks likely want either static or SSR docs, I wonder if linking these is ideal? But certainly NWTWWHB!

Reply:
I was thinking that the differences between the two would be instructive, so I wanted to link them so you could compare them. 

Do you think it's useful to have this equivalent SSR example to match the SSG one in the first place?